### PR TITLE
quick doc fixups

### PR DESCRIPTION
**Pull request by wozeparrot**
Followup to #900 with some additional fixes.

Gonna start documenting API after this.

Re stable diffusion: we should probably make sure that #834 isn't happening anymore.

### CODE REVIEW

File: docs/showcase.md [hunk 1]
```diff
@@ -53,7 +51,7 @@ python3 examples/stable_diffusion.py
 
 ### LLaMA
 
-You will need to download and put the weights into the [weights/LLaMA](/weightsLLaMA) directory, which may need to be created.
+You will need to download and put the weights into the `weights/LLaMA` directory, which may need to be created.
```
Comment (python273) [inline docs/showcase.md:54]: maybe link to llama-dl? 🤔 

https://github.com/shawwn/llama-dl/blob/56f50b96072f42fb2520b1ad5a1d6ef30351f23c/llama.sh

Comment (wozeparrot) [reply docs/showcase.md:54]: Are we legally allowed to do so? Just cuz of the license on LLaMA and that the original download links are behind the signup wall.

Comment (geohot) [reply docs/showcase.md:54]: We should probably switch to MPT or something by default. I don't think we should link to it.

Comment (wozeparrot) [reply docs/showcase.md:54]: There does exist OpenLLaMA: https://github.com/openlm-research/open_llama

Comment (geohot) [reply docs/showcase.md:54]: Is it as good as LLaMA? We really need a LM test suite. Unlocked that bounty.

* Get LLaMA 7B running with uint8 weights (leave float16 support too)
* Evaluate both the float16 one and uint8 one with ARC+HellaSwag+MMLU+TruthfulQA and confirm they match https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard

Comment (wozeparrot) [reply docs/showcase.md:54]: They claim that it has performance comparable to llama and better in some places.

I'll take a crack at getting the eval harness working. Do you want to depend on the harness or try to implement everything here as much as possible?

**Pull request merged**